drawBot/context/baseContext.py

```diff
@@ -620,6 +637,27 @@ def intersectionPoints(self, other=None):
             contours += other._contoursForBooleanOperations()
         return booleanOperations.getIntersections(contours)
 
+    def expandStroke(self, width, lineCap="round", lineJoin="round", miterLimit=10):
+        """
+        Returns a new bezier path with an expanded stroke around the bezier
+        path with a given width.
+
+        Optionally these options can be set:
+
+        * `lineCap`: Possible values are `"butt"`, `"square"` or `"round"`
+        * `lineJoin`: Possible values are `"bevel"`, `"miter"` or `"round"`
+        * `mitterLimit`: The miter limit to use for `"miter"` lineJoin option
```

Comment:
It says `mitterLimit` instead of `miterLimit` here.

Reply:
Also, needs some rewording:

```python
"""
Returns a new bezier path with an expanded stroke around the original path,
with a given `width`. Note: the new path will not contain the original path.

The following optional arguments are available with respect to line caps and joints:
* `lineCap`: Possible values are `"butt"`, `"square"` or `"round"`
* `lineJoin`: Possible values are `"bevel"`, `"miter"` or `"round"`
* `miterLimit`: The miter limit to use for `"miter"` lineJoin option
"""
```

Reply:
Sorry, "joins", not "joints"

Reply:
Thank you Just and sorry for my bad grammer! ( ͡~ ͜ʖ ͡°)